.npmignore

```diff
@@ -0,0 +1,91 @@
+# JS
+node_modules
+yarn.lock
```

Comment:
These two entries are excluded _automatically_ and to not need to be listed explicitly.